notebooks/community/vision/image_warehouse_sdk.ipynb

```diff
@@ -0,0 +1,1062 @@
+{
```

Comment:
<p>Headings are to be in Sentence Case:</p><p>Create corpus</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>hi Andrew, I change to "Create a Corpus" I think Corpus is a proper noun (corpus is a resource), similar to Index below. lmk if you think otherwise, thanks!</p>